lib/handbook/addon/docs/components/bs-alert/template.md

```diff
@@ -0,0 +1,9 @@
+# bs-alert
+
+Generates a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html).
+
+## info
+{{docs/components/bs-alert/demo-info}}
+
+## warning (not dismissible)
+{{docs/components/bs-alert/demo-warn-no-dismiss}}
```

Comment:
Maybe also add an example demonstrating `hidden` for completeness?

Something like:

```handlebars
{{#docs-demo as |demo|}}
    {{#demo.example name='bs-alert.hidden-danger.hbs'}}
            {{#bs-alert type='danger' dismissible=false hidden=hideAlert}}
                    <strong>Danger, Will Robinson!</strong>
            {{/bs-alert}}
    {{/demo.example}}

    {{#bs-button class='center-block m-b-lg' onClick=(action (mut hideAlert) (not hideAlert))}}
        set: hideAlert = {{not hideAlert}}
    {{/bs-button}}

    {{demo.snippet 'bs-alert.hidden-danger.hbs'}}
{{/docs-demo}}
```
would work nicely.